kv/txn_coord_sender.go

```diff
@@ -200,6 +200,7 @@ func (tc *TxnCoordSender) Close() {
 		close(txn.closer)
 	}
 	tc.txns = map[string]*txnMetadata{}
+	tc.wrapped.Close()
```

Comment:
Yes, this if fine. I did a review of all other senders. Change comment at line 163 in this file to read: "A TxnCoordSender should be closed when no longer in use via Close(), which also closes the wrapped sender supplied here."